src/main/java/com/cburch/logisim/vhdl/sim/VhdlSimulatorTclBinder.java

```diff
@@ -105,6 +105,9 @@ public void start() {
             // FIXME: hardcoded string
             final var errorMessage = new StringBuilder(
                     "You may disable VHDL simulation in the simulation menu if this occurs again\n\n");
+            
+            String msgCheckOnError = String.format("\"%s%s",
```

Comment:
`final var` instead of `String` please.

Reply:
Also, why isn't second "%s" (and also opening `\"` - I assume it's intentionally not closed) part of `VHDL_COMPILE_COMMAND`?

Reply:
`VHDL_COMPILE_COMMAND` is also used by
```
compFiles.append(VhdlSimConstants.VHDL_COMPILE_COMMAND)
            .append(componentName)
            .append(".vhdl")
            .append(System.getProperty("line.separator"));
```
So I didn't changed the function, I just replaced the string with the constant.
I could include the "%s" in the constant but then the said line should also be changed.
And yes, the not closed `\"` is intentional

Reply:
OK, thanks for clarification.